cli/command/system/info_test.go

```diff
@@ -21,8 +21,9 @@ func base64Decode(val string) []byte {
 	return decoded
 }
 
+var sampleID = "EKHL:QDUU:QZ7U:MKGD:VDXK:S27Q:GIPU:24B7:R7VT:DGN6:QCSF:2UBX"
```

Comment:
nit: `const`? (no blocker)

Reply:
This came from #1564 and I had the same feedback there, thought I'd fixed it there already! Will do it now (again?)